static/css/layout/v2.less

```diff
@@ -3,7 +3,7 @@
   box-sizing: border-box;
   max-width: 960px;
   width: 100%;
-  margin: 0 auto;
```

Comment:
will this affect every page?

Reply:
This will. I don't think this is intended or at least it should be scoped to iframes?
I think if you style floatform instead this will not be necessary.

![Screen Shot 2019-03-11 at 6 38 58 PM](https://user-images.githubusercontent.com/148752/54168682-00413000-442d-11e9-9593-3c00aca0420b.png)

Reply:
Yes that was unintentional. Thanks for reviewing. I will update the PR.